src/NetMQ/NetMQ.csproj

```diff
@@ -44,6 +44,9 @@
       <HintPath>..\packages\AsyncIO.0.1.17.0\lib\net40\AsyncIO.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="JetBrains.Annotations">
+      <HintPath>..\..\..\..\packages\JetBrains.Annotations.9.1.1\lib\net20\JetBrains.Annotations.dll</HintPath>
+    </Reference>
```

Comment:
I think it should be ..\packages\JetBrains.Annotations.9.1.1\lib\net20\JetBrains.Annotations.dll